lib/adoptoposs_web/templates/shared/footer.html.eex

```diff
@@ -12,7 +12,7 @@
     </div>
   </div>
 
-  <div class="text-center text-red-400 mt-4 mb-0">
+  <div class="text-center text-red-400 mt-4 mb-8">
```

Comment:
I for my part think I would just accept the cut-off footer behind the banner as long as the banner is visible. So, I'd remove the bottom margin again. Else, once you closed the banner, the footer looks a bit off in terms of distances/margins.